examples/Deploying-multi-stage-RecSys/02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb

```diff
@@ -61,6 +61,16 @@
     "### Import required libraries and functions"
```

Comment:
<p>can you pls remove this line? we added this into the first notebook, I assume we did not need it here anymore.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Seems safer to leave it here, just in case? That way you can run the notebooks in separate containers for now, but it'll still work once we shift to a single container.